ext/liquid_c/tokenizer.c

```diff
@@ -90,10 +99,13 @@ void tokenizer_next(tokenizer_t *tokenizer, token_t *token)
                 if (c != '}')
                     continue;
                 token->type = TOKEN_TAG;
+                if(token->str[cursor - tokenizer->cursor - 3] == '-')
```

Comment:
Why not check `cursor[-3]` instead of `token->str[cursor - tokenizer->cursor - 3]`